modules/ROOT/pages/starting-and-stopping-mule-esb.adoc

```diff
@@ -0,0 +1,201 @@
+= Starting and Stopping Mule
```

Comment:
This article has almost the same information that /runtime-installation-task.adoc, is there any reason to have both?